### ISSUE TITLE
2.7.3 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2679]
- #2641 
- #2651
- #2226 
- #2657 
- #2664 
- #2620
- #2678 

## Pick list
```
git cherry-pick 7fff9db9f973e317ed807e9b53907aa034427738
git cherry-pick 7dc7e97fbe97935fd4550dba8bd2e50c99f0cc80
git cherry-pick 224e4da9e0d03f31539422eb4ce81c317af1276c
git cherry-pick e6ac4323528f583a5fb8ae87eaf9b455bff7366a
git cherry-pick 486587f192e1e658c59d55d046c76ccb11a4728c
git cherry-pick 7e8cb5718768b99474a79e215b65eb467cbdbe9a
git cherry-pick f5725025f39b93a769d1c8aa4a8a5115308504b7
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.7.2..HEAD --pretty='format:- %s'`
```
- Closes #2677: Fix broken Google Tag config upgrade. (#2678)
- Fixes #2619 News export API empty value handling (#2620)
- Closes #2663 - Allow multiple access schemes with Workbench Access (#2664)
- Closes #2656: Switch back to d.o. version of background_image_formatter module. (#2657)
- Fixes #2212 Adding a README for our add-permissions drush command (#2226)
- Closes #2644: Marketing Cloud module initial feedback (#2651)
- Closes #2639: Fix deprecation warning for cards with link title value and no link URL value. (#2641)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.3
